File: src/2798-Number-of-Employees-Who-Met-the-Target/number_of_employees_who_met_the_target.py
from typing import List

class Solution:
    def numberOfEmployeesWhoMetTarget(self, hours: List[int], target: int) -> int:
        ans: int = 0

        for _ in range(len(hours)):
            if hours[_] >= target:
                ans += 1
        
        return ans

if __name__ == "__main__":
    hours: List[int] = [0,1,2,3,4]
    target: int = 2

    solution = Solution()
    
    print(solution.numberOfEmployeesWhoMetTarget(hours, target))

File: src/2798-Number-of-Employees-Who-Met-the-Target/number_of_employees_who_met_the_target.cpp
#include <iostream>
#include <vector>

class Solution {
public:
    int numberOfEmployeesWhoMetTarget(std::vector<int>& hours, int target) {
        int ans = 0;

        for (int i = 0; i < hours.size(); i++) {
            if (hours[i] >= target) ans += 1;
        }

        return ans;
    }
};


int main() {
    std::vector<int> hours = {0,1,2,3,4};
    int target = 2;

    Solution solution;

    std::cout << solution.numberOfEmployeesWhoMetTarget(hours, target);
    
    return 0;
}